wellcomeml/ml/bert_semantic_equivalence.py

```diff
@@ -181,7 +188,9 @@ def fit(self, X, y, random_state=None, epochs=3, metrics=[], **kwargs):
             )
             self.valid_dataset = self.valid_dataset.batch(self.eval_batch_size)
 
-            self._compile_model()
+            strategy = self._get_distributed_strategy()
```

Comment:
Sorry for that, but I spoke too soon. Just tested ant this won't work, because you need to create the model with this strategy.

So you need to put `strategy = self._get_distributed_strategy()` before `self.initialise_models()` (L172).

Reply:
And than that line will become

```python
with strategy.scope(): 
     self.initialise_models()
```

Reply:
makes sense, i did that when loading but not here. ok will change